packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -131,8 +131,8 @@ export class ProblemWidget extends TreeWidget {
     protected decorateMarkerFileNode(node: MarkerInfoNode): React.ReactNode {
         return <div className='markerFileNode'>
             <div className={(node.icon || '') + ' file-icon'}></div>
-            <div>{node.name}</div>
-            <div className='path'>{node.description || ''}</div>
+            <div title={node.name} className='name'>{node.name}</div>
```

Comment:
is `name` the name of the css class? just asking ... as i did not find it in `.../theia/packages/markers/src/browser/style/index.css`